tests/widget/inline/widget-inline.register.test.js

```diff
@@ -3,38 +3,38 @@ import { v4 as uuid } from 'uuid';
 
 import { asyncEvents, verifyUserInfo } from '../../utilities/async-events.js';
 
-test('Inline widget with user registration', async ({ page }) => {
-  const { clickButton, navigate } = asyncEvents(page);
+// test('Inline widget with user registration', async ({ page }) => {
+//   const { clickButton, navigate } = asyncEvents(page);
 
-  await navigate('widget/inline?journey=TEST_Registration');
+//   await navigate('widget/inline?journey=TEST_Registration');
 
-  await page.getByLabel('Username').fill(uuid());
-  await page.getByLabel('First Name').fill('Demo');
-  await page.getByLabel('Last Name').fill('User');
-  await page.getByLabel('Email Address').fill('test@auto.com');
+//   await page.getByLabel('Username').fill(uuid());
+//   await page.getByLabel('First Name').fill('Demo');
+//   await page.getByLabel('Last Name').fill('User');
+//   await page.getByLabel('Email Address').fill('test@auto.com');
 
-  // Intentionally entire password that fails validation
-  await page.getByLabel('Password').fill('willfail');
+//   // Intentionally entire password that fails validation
+//   await page.getByLabel('Password').fill('willfail');
 
-  await page
-    .getByLabel('Select a security question')
-    .selectOption({ label: `What's your favorite color?` });
-  await page.getByLabel('Security Answer').fill('Red');
-  // `getByLabel()` does not work, likely due to the `<span>` elements (?)
-  await page.getByText('Please accept our Terms & Conditions').click();
+//   await page
+//     .getByLabel('Select a security question')
+//     .selectOption({ label: `What's your favorite color?` });
+//   await page.getByLabel('Security Answer').fill('Red');
+//   // `getByLabel()` does not work, likely due to the `<span>` elements (?)
+//   await page.getByText('Please accept our Terms & Conditions').click();
 
-  await clickButton('Register', '/authenticate');
+//   await clickButton('Register', '/authenticate');
 
-  await expect(page.getByText('Password requirements')).toBeVisible();
-  await expect(page.getByText('At least 1 number(s)')).toBeVisible();
-  await expect(page.getByText('At least 1 uppercase')).toBeVisible();
-  await expect(page.getByText('At least 1 lowercase')).toBeVisible();
-  await expect(page.getByText('At least 1 symbol(s)')).toBeVisible();
+//   await expect(page.getByText('Password requirements')).toBeVisible();
+//   await expect(page.getByText('At least 1 number(s)')).toBeVisible();
+//   await expect(page.getByText('At least 1 uppercase')).toBeVisible();
+//   await expect(page.getByText('At least 1 lowercase')).toBeVisible();
+//   await expect(page.getByText('At least 1 symbol(s)')).toBeVisible();
```

Comment:
This isn't blocking as the fix for the test is in this PR: https://github.com/ForgeRock/forgerock-web-login-framework/pull/94. But, I would only comment out these 4 lines as they are the only issue at this moment, not the entire test itself :)